### ISSUE TITLE
App crashes in tablet on tap on camera.

### ISSUE THREAD

**Issue by mohitmanuja** [fossasia/phimpme-android#415]
**Actual Behaviour**

App is crashing on camera layout.

**Expected Behaviour**

Should be run on all devices.

**Steps to reproduce it**

Open app in tablet and than click on the camera button.

**LogCat for the issue**

Provide logs for the crash here

FATAL EXCEPTION: main
                                                                 Process: vn.mbm.phimp.me, PID: 18183
                                                                 java.lang.NullPointerException
                                                                     at vn.mbm.phimp.me.Camera2.setLayout(Camera2.java:386)
                                                                     at vn.mbm.phimp.me.Camera2.onCreateView(Camera2.java:166)
                                                                     at android.support.v4.app.Fragment.performCreateView(Fragment.java:2087)
                                                                     at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1113)
                                                                     at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1295)
                                                                     at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:801)
                                                                     at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1682)
                                                                     at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:541)
                                                                     at android.os.Handler.handleCallback(Handler.java:755)
                                                                     at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                     at android.os.Looper.loop(Looper.java:145)
                                                                     at android.app.ActivityThread.main(ActivityThread.java:5266)
                                                                     at java.lang.reflect.Method.invokeNative(Native Method)
                                                                     at java.lang.reflect.Method.invoke(Method.java:515)
                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:826)
                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:642)
                                                                     at dalvik.system.NativeStart.main(Native Method)

**Would you like to work on the issue?**

Yes.

**Comment by mohitmanuja** [comment]
Currently, I am putting it on hold will solve after #421.
